src/runtime/test/vdom-relocation.spec.tsx

```diff
@@ -2,7 +2,7 @@ import { Component, Listen, State, h } from '@stencil/core';
 import { newSpecPage } from '@stencil/core/testing';
 
 describe('vdom-relocation', () => {
-  it('', async () => {
```

Comment:
😱

Reply:
yeah...this and the duplicate tests I was glad to find them at least!